packages/sdk/src/protocols/base_protocol.ts

```diff
@@ -197,20 +193,20 @@ export class BaseProtocolSDK implements IBaseProtocolSDK {
     try {
       let newPeers = await this.core.getPeers({
         maxBootstrapPeers: 0,
-        numPeers: numPeers
+        numPeers: 0
```

Comment:
then I would say it is a misleading API definition. We literally say here: get me peers with 0 peers and 0 bootstrap peers. 🙃

Reply:
but why do we want to get *all* peers in the first place and not do as before to get only needed amount? 

as for API I think if it has no parameters - it would return all peers
`this.getPeers({ bootstrap: 0 })`